server/api/region.go

```diff
@@ -130,6 +131,47 @@ func (h *regionsHandler) CheckRegionsReplicated(w http.ResponseWriter, r *http.R
 	h.rd.JSON(w, http.StatusOK, state)
 }
 
+// @Tags     region
+// @Summary
+// @Accept   json
+// @Param    start_key  body  string  true  "Regions start key, hex encoded"
+// @Param    end_key    body  string  true  "Regions end key, hex encoded"
+// @Param    batch_size  body  string  true  "Maximum operators scheduled in one"
+// @Param    timeout    body  string  true  "Timeout time in milliseconds"
+// @Param    required_labels    body  array  true  "Only the stores with these labels would be scheduled"
```

Comment:
```suggestion
```
User only specify the key-range, this API would follow the `<region,zone,host>` constraint for rebalancing the peer among stores

Reply:
Yes, maybe we can follow the pattern of scatter region